Sources/Extensions/UIKit/UILayoutPriorityExtensions.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  UILayoutPriorityExtensions.swift
+//  SwifterSwift
+//
+//  Created by diamantidis on 8/19/18.
+//  Copyright © 2018 SwifterSwift. All rights reserved.
+//
+
+#if os(iOS) || os(tvOS)
```

Comment:
Hey @diamantidis 
This platform check can be replaced by a more simple `#if canImport(UIKit)` ?

Reply:
Hi @LucianoPAlmeida !!!
`UIKit` is available on `iOS`, `tvOS` and `watchOS`, whereas `UILayoutPriority` is available on `iOS` and `tvOS`. So changing to `#if canImport(UIKit)` will make the build for `watchOS` fail

Reply:
Hey @diamantidis :)) 
UIKit is available on WatchOS 🤔? That is weird 😆
But not problems with the platform checks :) 
Is quit the same result at the end of the day 👍

Reply:
Yes @LucianoPAlmeida, it seems so. Initially I used `#if canImport(UIKit)`, but travis failed while building `watchOS`.
The `UIKit` documentation also mention about SDK `watchOS 2.0+`.
https://developer.apple.com/documentation/uikit

Reply:
Humm That makes sense :))
Thank's @diamantidis 👍